iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/ReactiveCapabilityCurveImpl.java

```diff
@@ -30,6 +30,9 @@ static class PointImpl implements Point {
         private double maxQ;
 
         PointImpl(double p, double minQ, double maxQ) {
+            if (minQ > maxQ) {
+                throw new IllegalStateException("minQ should be inferior or equal to maxQ");
+            }
```

Comment:
There's a related TODO written in 2017 (to plan it for IIDM v1.1!!) in the corresponding adder, and a `@Disabled` in a unit test in AbstractReactiveCapabilityCurveTest. Could you remove them?

This should be only activated for next IIDM version. There should be a check in the importer based on the IIDM version to flip or not.